spec/javascripts/NewService/FormTemplate.spec.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import Enzyme, {shallow} from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
+
+import {FormTemplate} from 'NewService'
+import * as utils from 'utilities/utils'
+utils.CSRFToken = jest.fn(() => '')
+
+Enzyme.configure({adapter: new Adapter()})
+
+const serviceDiscoveryProps = {
+  isFormDiscovery: true,
+  isVisible: true,
+  formActionPath: 'my-path',
+  namespaces: ['one', 'two', 'three'],
+  services: ['four', 'five', 'six'],
+  selectedNamespace: 'selectedNamespace',
+  onChangeNamespaces: jest.fn()
+}
+const newServiceProps = {
+  isFormDiscovery: false,
+  isVisible: true,
+  formActionPath: 'my-other-path'
+}
+
+describe('<FormTemplate/>', () => {
```

Comment:
There is no point in declaring a describe block for the file being tested, it's redundant. `describe` is meant to _describe_ different scenarios like _when isVisible is true_ for instance.